.github/workflows/gh-build-and-test.yml

```diff
@@ -280,6 +283,13 @@ jobs:
           host-platform: ${{ matrix.host-platform }}
           cuda-version: ${{ matrix.cuda-version }}
 
+      - name: Install dependencies
```

Comment:
TBH I don't want to install any build tools in the test pipeline, but it seems to be the easiest path...